traffic_ops/traffic_ops_golang/steeringtargets/steeringtargets.go

```diff
@@ -160,6 +160,10 @@ func read(tx *sqlx.Tx, parameters map[string]string, user *auth.CurrentUser) ([]
 }
 
 func (st *TOSteeringTargetV11) Create() (error, error, int) {
+	err := st.Validate()
```

Comment:
So the `Validate` method does not need to be called directly here and below, because it gets called by the `api.CreateHandler` and `api.UpdateHandler` methods via `decodeAndValidateRequestBody` in api/shared_handlers.go. By the time the `Create`/`Update` method is called on this object, its `Validate` method has already been called.

Reply:
Aah got you, fixed now.